railties/lib/rails/generators/app_base.rb

```diff
@@ -109,6 +109,9 @@ def self.add_shared_options_for(name)
         class_option :skip_ci,             type: :boolean, default: nil,
                                            desc: "Skip GitHub CI files"
 
+        class_option :devcontainer,        type: :boolean, default: true,
```

Comment:
Should we not follow the naming conventions here?

> This would give us `--no-devcontainer`. We should do the same on the app generator as well

If `--no-` is the default, all others should follow this naming too?